bitnami/airflow/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: airflow
-version: 6.6.0
+version: 6.6.2
```

Comment:
I would say this is a minor version rather than a patch, what do you think?

Reply:
@FraPazGal I agree.